lib/cli.js

```diff
@@ -252,6 +252,12 @@ internals.options = function () {
             description: 'minimum plan threshold to apply to all tests that don\'t define any plan',
             default: null
         },
+        dep: {
```

Comment:
The nodejs, mocha, and ava CLIs all call this option `require`.  Perhaps for consistency we may call it `require` with a short alias of `req` (since `r` and `R` are taken)?